lib/libshout/CMakeLists.txt

```diff
@@ -3,11 +3,14 @@ project(shout_mixxx C)
 
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DHAVE_CONFIG_H -Wall -ffast-math -pthread -g -O2")
  
+find_package(OpenSSL)
+
 include_directories(
   .
   src
   include
   src/common
+  "${OPENSSL_INCLUDE_DIR}"
```

Comment:
Do you know why this not works?  
```
"${OpenSSL_INCLUDE_DIRS}" ? 
```

I think instead plain "ssl crypto" in 
```
target_link_libraries(shout_mixxx ogg vorbis theora speex ssl crypto)
``` 
we can use uses: 
${OPENSSL_LIBRARIES}

Please add this as well.

Reply:
> Do you know why this not works?  
> ```
> "${OpenSSL_INCLUDE_DIRS}" ? 
> ```

Backwards-compatibility to older cmake versions where the preferred naming scheme was not finalized.

> I think instead plain "ssl crypto" in 
> ```
> target_link_libraries(shout_mixxx ogg vorbis theora speex ssl crypto)
> ``` 
> we can use uses: 
> ${OPENSSL_LIBRARIES}

We should use the imported targets (`OpenSSL::SSL` and `OpelSSL::Crypto`) which takes care of linking the appropriate libraries and adding the necessary include dirs.

https://cmake.org/cmake/help/latest/module/FindOpenSSL.html

Reply:
ping

Reply:
Do you need any action from me?

Reply:
Yes, it would be nice if you can fix what we have commented. 
If you are out of time for this, I can take over, no problem just give a hint.

Reply:
I have made the changes requested.